recipes/meta/Firefox.yml

```diff
@@ -0,0 +1,22 @@
+app: Firefox
+
+ingredients:
+  script:
+    - DLD=$(wget -q "https://www.mozilla.org/en-US/firefox/all/" -O - | grep -E "os=linux64&amp;lang=en-US" | cut -d'"' -f2)
+    - wget -c $DLD --trust-server-names
+    - echo $DLD | cut -d- -f2 > VERSION
+
+script:
+  - tar xfj ../firefox*.tar.bz2
```

Comment:
Please do the extraction of the ingredients in the script section of the ingredients section